Gruntfile.package.js

```diff
@@ -46,7 +46,7 @@ module.exports = function(grunt) {
       },
       snapshots: {
         src: [
-          './packages/<%= package %>/**/__snapshots__'
+          './packages/<%= package %>/src/**/__snapshots__'
```

Comment:
I went ahead and added the `/tests/` folder as well just in case another project uses that folder for snapshots:
https://github.com/adamweeks/spark-js-sdk/blob/73665d8e9e00cea7358721fabab2f021f38adaa3/Gruntfile.package.js#L50